content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -355,6 +355,17 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### volumes.kubernetes.io/controller-managed-attach-detach
+
+Used on: Node
+
+If a node has set the annotation `volumes.kubernetes.io/controller-managed-attach-detach`
+on itself, then its storage attach and detach operations are being managed
+by the attach/detach controller running in kube-controller-manager.
```

Comment:
```suggestion
by the _volume attach/detach_
{{< glossary_tooltip text="controller" term_id="controller" >}} running within the
{{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
```

Some people call it the “attach/detach controller”; I prefer to include “volume” for clarity.

Reply:
Addressed the changes